TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Constants.java

```diff
@@ -88,9 +88,9 @@ public static abstract class IntakeConstants {
         public static final double INTAKE_WRIST_SERVO_UP_POSITION = 0;
         public static final double INTAKE_WRIST_SERVO_DOWN_POSITION = 180;
 
-        public static int MANUAL_CONTROL_RATE = 2; //add final later
+        public static int MANUAL_CONTROL_RATE = 25; //add final later
 
-        public static final int MAXIMUM_SLIDE_POS = 2800;
+        public static final int MAXIMUM_SLIDE_POS = 2900;
```

Comment:
Are you sure it can go this far safely. I'm trusting you but it's a bit worrisome, I wouldn't want to break the robot...